spec_util/meld_test.go

```diff
@@ -614,6 +614,30 @@ func TestMeldData(t *testing.T) {
 				PotentialConflict: true,
 			}),
 		},
+		{
+			name: "merging oneof with unformatted primitive",
+			left: wrapOneOf(&pb.OneOf{
+				Options: map[string]*pb.Data{
+					"1": wrapPrim(&pb.Primitive{
+						Value:      &pb.Primitive_Uint32Value{Uint32Value: &pb.Uint32{}},
+						FormatKind: "datetime",
+						Formats:    map[string]bool{"timestamp_seconds_since_epoch": true},
+					}),
+					"2": wrapPrim(&pb.Primitive{
+						Value:      &pb.Primitive_Uint64Value{Uint64Value: &pb.Uint64{}},
+						FormatKind: "unique_id",
+						Formats:    map[string]bool{"integer_id": true},
+					}),
```

Comment:
I would like to see the case where there is an incompatible type in the list as well, so that (string | datetime | unique_id) with (int) becomes (string | int).  i.e., where the new len(Options) == 0 branch is not taken, but some options have been deleted anyway.

Reply:
In 4bcf3e4.